python/paddle/fluid/distribute_transpiler.py

```diff
@@ -252,12 +315,114 @@ def transpile(self,
                 outputs={"Out": [orig_param]},
                 attrs={"axis": 0})
 
+        if self.has_distributed_lookup_table:
```

Comment:
Can we move these following code into an  independent function?

Reply:
done